tools/add-feed.js

```diff
@@ -30,7 +30,7 @@ async function add() {
       attempts: process.env.FEED_QUEUE_ATTEMPTS || 8,
       backoff: {
         type: 'exponential',
-        delay: process.env.FEED_QUEUE_DELAY_MS || 60 * 1000,
+        delay: (process.env.FEED_QUEUE_DELAY_MS || 60) * 1000,
```

Comment:
I think the value in `FEED_QUEUE_DELAY_MS` is already in milliseconds (notice the MS at the end), so I think this should be:
`delay: process.env.FEED_QUEUE_DELAY_MS || (60 * 1000)`, which might not even need
(would have to check operator priority).

Reply:
Good call; reverted this changes: https://github.com/Seneca-CDOT/telescope/pull/550/commits/68a4ad098973259828fbccbecef44a4aaebbda86